### DISCUSSION TITLE
Module not found: Error: Can't resolve '../api/ServiceData.json' in '/vercel/path0/src/components'

### DISCUSSION THREAD

**Discussion by sameeeyyy** [vercel/community#3030] [Help]
### Summary

During the deployment of a project on Vercel, you encountered an error related to a missing module. Specifically, the error message states that the module '../api/ServiceData.json' could not be resolved in the '/vercel/path0/src/components' directory. As a result, the build process failed, leading to an exit code of 1. To resolve this issue, you need assistance from the Vercel support team. To expedite the troubleshooting process, provide the following information when submitting a ticket.

### Steps to Reproduce

1. Attempt to deploy the project on Vercel.
2. Observe the error message during the deployment process.

> Failed to compile.
> Module not found: Error: Can't resolve '../api/ServiceData.json' in '/vercel/path0/src/components'
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> Error: Command "yarn run build" exited with 1
> BUILD_UTILS_SPAWN_1: Command "yarn run build" exited with 1

**Comment by amyegan** [comment]
If you share the unique deployment URL or deployment ID, then I can look into the logs for more info.
